### DISCUSSION TITLE
Settings / Parameters documentation

### DISCUSSION THREAD

**Discussion by Kinrob** [openai/whisper#2108] [Q&A]
Looking to find information regarding the parameters i.e "--threads THREADS"

Researched the Q&A and Reddit, really wasn't seeing a ton, I'm sure I missed it. If not, I do understand some of it is self-explanatory i.e. --output format, and also do understand research is required with parameters similar to temperature, beam size, compression ratio, etc. I don't necessarily need that, just want to know what the parameters do and what values they'll accept. 

 [--temperature TEMPERATURE] [--best_of BEST_OF] [--beam_size BEAM_SIZE] [--patience PATIENCE]
               [--length_penalty LENGTH_PENALTY] [--suppress_tokens SUPPRESS_TOKENS] [--initial_prompt INITIAL_PROMPT]
               [--condition_on_previous_text CONDITION_ON_PREVIOUS_TEXT] [--fp16 FP16]
               [--temperature_increment_on_fallback TEMPERATURE_INCREMENT_ON_FALLBACK]
               [--compression_ratio_threshold COMPRESSION_RATIO_THRESHOLD] [--logprob_threshold LOGPROB_THRESHOLD]
               [--no_speech_threshold NO_SPEECH_THRESHOLD] [--word_timestamps WORD_TIMESTAMPS]
               [--prepend_punctuations PREPEND_PUNCTUATIONS] [--append_punctuations APPEND_PUNCTUATIONS]
               [--highlight_words HIGHLIGHT_WORDS] [--max_line_width MAX_LINE_WIDTH] [--max_line_count MAX_LINE_COUNT]
               [--max_words_per_line MAX_WORDS_PER_LINE] [--threads THREADS]
               audio [audio ...]

**Comment by glangford** [comment]
`whisper --help` and search past discussions to get clarification.
